components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -123,19 +123,19 @@ function InputBox(props: InputBoxProps) {
 
   React.useEffect(() => {
     // Update the height of the attachment wrapper when
-    // pendingMessageImages changes.
-    if (props.context?.pendingMessageImages.length > 0) {
+    // pendingMessageFiles changes.
+    if (props.context?.pendingMessageFiles.length > 0) {
```

Comment:
I don't think context can be falsey here, or the rendering will crash (as we access properties off context)
```suggestion
    if (props.context.pendingMessageFiles.length > 0) {
```

Reply:
fixed in 53509362ade970cc99956e5006a8665150ca4136